spec/features/auth/login_feature_spec.rb

```diff
@@ -22,26 +22,26 @@
     expect(page).to have_content("Configuration")
   end
 
-  it "Wrong password results in an error message" do
-    pending("fix the validations")
+  it "shows an error message when using invalid credentials" do
+    # pending("fix the validations")
     fill_in "user_email", with: "foo"
     fill_in "user_password", with: "bar"
-    find("input[type=submit]", match: :first).click
+    click_button("Log in")
```

Comment:
I think here is mistake? Space between "Log in"

Reply:
No, it's "Log in".

https://github.com/kubic-project/velum/blob/master/spec/features/auth/login_feature_spec.rb#L20

https://github.com/kubic-project/velum/blob/master/app/views/devise/registrations/new.html.slim#L12

Reply:
"Log in" is also more correct from an English point of view :)